pkgs/tools/system/snooze/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchFromGitHub }:
+stdenv.mkDerivation rec {
+  pname = "snooze";
+  version = "0.4";
+  src = fetchFromGitHub {
+    owner = "leahneukirchen";
+    repo = "snooze";
+    rev = "v${version}";
+    sha256 = "0a114brvvjf6vl7grviv0gd6gmikr447m8kq1wilp4yj51sfyxa9";
+  };
+  makeFlags = [ "DESTDIR=$(out)" "PREFIX=/" ];
+
+  meta = with stdenv.lib; {
+    description =
+      "Tool for waiting until a particular time and then running a command.";
+    maintainers = with maintainers; [ kaction ];
+    license = licenses.publicDomain;
```

Comment:
This work appears to be dual licensed public domain and CC0. I think CC0 is preferred because some jurisdictions don't recognize public domain.

Reply:
Ok, updated.